backend/README.md

```diff
@@ -82,3 +82,17 @@ pytest
 
 More documentation on `pytest` can be found [here](https://docs.pytest.org/en/8.0.x/).
 
+## Contributing
+
+### Type errors
+
+The vscode extension [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) can be used to highlight basic type errors in your python code. To set this up do the following:
+
+- Install the extension
+- Add the following to your `../.vscode/settings.json` file:
```

Comment:
A bit ambiguous, my `.vscode` directory is in my user home directory, specifying `..` doesn't get me there on my system, maybe `$HOME/.vscode/settings.json` would be a better pointer.

Reply:
you can have two `settings.json` , your user one as you say is at `$HOME/.vscode` , but you can also have a workspace one, which just applies to the current project , which lives under the `<project_root>/.vscode` . I kept it as the workspace one here as thought is more relavant, but maybe I should mention both options? What do you think?

Reply:
Ah, fair enough, I only have the one in $HOME/.vscode. I didnt realise that you could have a workspace one, makes sense though.